### DISCUSSION TITLE
Show External IP only in callbacks page (avoid local domain ips)

### DISCUSSION THREAD

**Discussion by el3ctr0wqw1** [its-a-feature/Mythic#434] [General]
As shown in title , can anyone suggest me a way to do so ? i need to know external ips to facilitate the machines identification.

**Comment by its-a-feature** [comment]
On the active callbacks page where you see your table of active callbacks, you can right click the headers of that table and select "Show/Hide Columns". From there you can select to show the tracked external IP and hide any other fields you don't want to see. That should allow you to see the collected external IP for the callbacks

**Reply by el3ctr0wqw1** [reply]
Thank you so much! i would never guessed
